test/cases/parsing/optional-chaining/index.js

```diff
@@ -0,0 +1,5 @@
+it("should handle optional members", () => {
+	expect(
+		module.hot?.accept((() => {throw new Error("fail")})())
```

Comment:
Should I move this to `configCases` since this does not work with `TestCasesHot`?